HelpMyStreetFE/HelpMyStreetFE/Services/AuthService.cs

```diff
@@ -56,5 +54,27 @@ public async Task LoginWithTokenAsync(string token, HttpContext httpContext)
             await httpContext.SignInAsync(CookieAuthenticationDefaults.AuthenticationScheme,
                 new ClaimsPrincipal(identity));
         }
+
+        private string getGoogleCredentialJson()
+        {
+            try
+            {
+                var firebaseCredentials = _configuration["Firebase:Credentials"];
+
+                if (firebaseCredentials == string.Empty)
+                {
+                    throw new Exception("Firebase cedentials missing");
+                }
+
+                var credential = File.ReadAllText(firebaseCredentials);
```

Comment:
Is your `Firebase:Credentials` key still pointing at a file? The key for the firebase credentials should now be a JSON string in the appsettings.Development.json file. This is so that it can be stringified in to an environment variable in the deployment. I updated the README but should probably put something in slack, I'll send you the appsettings.Development.json file I've got